server/meshmodel/k8svault-controller/relationships/edge_relationship.json

```diff
@@ -0,0 +1,61 @@
+{
+  "apiVersion": "vault.infra.doodle.com/v1beta1",
+  "kind": "Edge",
+  "metadata": {
+    "description": ""
+  "model": {
+    "name": "k8svault-controller",
+    "version": "0.1.2",
```

Comment:
Will this relationship work for `version: "0.1.2"`, but not for `version: "1.1.155"` ?

Reply:
If it works for both use `version: "*"`.